charts/kubefed/README.md

```diff
@@ -25,38 +25,8 @@ group](https://groups.google.com/forum/#!forum/kubernetes-sig-multicluster).
 ## Prerequisites
 
 - Kubernetes 1.13+
-- Helm 2.10+
+- Helm 3.1.2+
```

Comment:
do we still support k8s 1.13?

Reply:
I feel like this should just be `Helm 3`, I don't see a need for adding the minor and bugfix semver.

Reply:
Kubernetes is 1.13+, so all versions newer are covered.
For Helm I changed to 3.2+ to the --create-namespace option